### PR TITLE
Document add_row()

### PR DESCRIPTION
**Pull request by niansa**

### CODE REVIEW

**Comment by niansa** [comment]
I hope that's enough and I follow the code style close enough (code style isn't my thing lol)

**Comment by niansa** [comment]
Might as well document how to then access the text from the second row

File: docpages/example_programs/interactions_and_components/modal_dialog_interactions.md [hunk 1]
```diff
@@ -29,9 +29,21 @@ int main(int argc, char const *argv[])
...
+				set_label("Short type rammel").
 				set_id("field_id").
 				set_type(dpp::cot_text).
+				set_placeholder("gumd").
+				set_min_length(5).
+				set_max_length(50).
+				set_text_style(dpp::text_short)
+			);
+			/* Add another text component in the next row, as required by Discord */
+			embed.add_row();
```
Comment (braindigitalis) [inline docpages/example_programs/interactions_and_components/modal_dialog_interactions.md:41]: should be modal.add_row() not embed.add_row() surely?

**Review by braindigitalis — CHANGES_REQUESTED** [review]

File: docpages/example_programs/interactions_and_components/modal_dialog_interactions.md (hunk 1 above)
Comment (niansa) [reply docpages/example_programs/interactions_and_components/modal_dialog_interactions.md]: solved.

**Review by braindigitalis — APPROVED** [review]

**Pull request merged**